go.mod

```diff
@@ -6,6 +6,7 @@ require (
 	github.com/crossplane/crossplane-runtime v0.14.0
 	github.com/crossplane/crossplane-tools v0.0.0-20201201125637-9ddc70edfd0d
 	github.com/google/go-cmp v0.5.2
+	github.com/hashicorp/go-getter v1.4.0
```

Comment:
go-getter/v2 should have fewer dependencies

Reply:
@k0da i took the version they use in terraform cli